rpc/block_modify.c

```diff
@@ -745,11 +742,36 @@ blockModifySizeCliFormatResponse(blockModifySizeCli *blk, blockModifySize *mobj,
   GB_FREE(tmp3);
 
   /*catch all*/
-  if (!reply->out) {
-    blockFormatErrorResponse(MODIFY_SIZE_SRV, blk->json_resp, errCode,
-                             GB_DEFAULT_ERRMSG, reply);
+  if (errCode && !errMsg) {
+    GB_ASPRINTF (&errMsg, "block volume resize failed:");
+    for (i = 0; i < info->nhosts; i++) {
+      switch (blockMetaStatusEnumParse(info->list[i]->status)) {
+      case GB_RS_FAIL:
+      case GB_RS_INPROGRESS:
+        GB_FREE(hr_size);
+        hr_size = blockInfoGetCurrentSizeOfNode(blk->block_name, info, info->list[i]->addr);
```

Comment:
May be a log here when hr_size return null would be good right @lxbsz

Reply:
Yeah, make sense.